packages/fluent-theme/src/components/activity/CopilotMessageHeader.tsx

```diff
@@ -0,0 +1,40 @@
+import { WebChatActivity, hooks } from 'botframework-webchat-component';
+import React, { memo, useMemo, type CSSProperties } from 'react';
+import { useStyles } from '../../styles';
+import styles from './CopilotMessageHeader.module.css';
+
+const { useStyleOptions, useLocalizer } = hooks;
+
+function CopilotMessageHeader({ activity }: Readonly<{ activity?: WebChatActivity | undefined }>) {
+  const [{ botAvatarImage, botAvatarBackgroundColor, botTitle }] = useStyleOptions();
+  const classNames = useStyles(styles);
+  const localize = useLocalizer();
+  // TODO: how we determine the activity has ai-generated content
```

Comment:
Something from here, https://github.com/microsoft/BotFramework-WebChat/pull/5259/files#diff-e850c1bed138c1b7171eba0b4e8d96154daf2aa682c8fa231212b3c87d9821e7.

`getMessageEntity(activity)?.keywords.includes('AIGeneratedContent')`

We can update this when the other PR is in.

Reply:
Yea, let's work on it a bit later